tendrl/commons/flows/import_cluster/gluster_help.py

```diff
@@ -115,6 +115,7 @@ def import_gluster(parameters):
                    "log_level": "DEBUG",
                    "logging_socket_path": "/var/run/tendrl/message.sock",
                    "sync_interval": 180,
+                   "ttl_delta": 600,
```

Comment:
This is not required, I was looking at ttl_delta being calculated by Tendrl during gluster sync 

ttl_delta = count(vols) X 2